### PR TITLE
Random_list, sum & average

### PR DESCRIPTION
**Pull request by alinadmytriv**
Create a random list of X elements, iterate through list of randoms and calculate its sum and average.

### CODE REVIEW

File: laboratorna1/random_list.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+x = int(input("Enter the number of array elements: "))
+y = int(input("Enter number for the beginning: "))
+z = int(input("Enter number for the ending: "))
+
+random_list = []
+for i in range(x):
+    random_list.append(random.randint(y, z))
+
+print(random_list)
```
Comment (polyakovartem) [inline laboratorna1/random_list.py:11]: same comment as for lab2

File: laboratorna1/random_list.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import random
+
+x = int(input("Enter the number of array elements: "))
+y = int(input("Enter number for the beginning: "))
+z = int(input("Enter number for the ending: "))
```
Comment (polyakovartem) [inline laboratorna1/random_list.py:5]: please, create the separate function for this

**Review by polyakovartem — CHANGES_REQUESTED** [review]

File: lab1/random_list0.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# find the average of the list
+def average(list1):
+    return summary(list1)/len(list1)
+
+
+# the main function which ask to input details about the list
+def main():
+    x = int(input("Enter the number of array elements: "))
+    y = int(input("Enter number for the beginning: "))
+    z = int(input("Enter number for the ending: "))
```
Comment (polyakovartem) [inline lab1/random_list0.py:26]: move input into separate function

File: lab1/random_list0.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    l = create_list(x, y, z)
+    print("Random list - " + str(l))
+    print("Sum - " + str(summary(l)))
+    print("Average - " + str(average(l)))
```
Comment (polyakovartem) [inline lab1/random_list0.py:30]: move result printing into separate function

File: lab1/random_list0.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import random
+
+
+# create a random list
+def create_list(x0, y0, z0):
```
Comment (polyakovartem) [inline lab1/random_list0.py:5]: please use informative parameter names

**Review by polyakovartem — CHANGES_REQUESTED** [review]
please do not create a new file, make all the modification in an old one

**Review by polyakovartem — CHANGES_REQUESTED** [review]
why did you create a new file one more time?

**Comment by alinadmytriv** [comment]
I am just trying to understad how to use the github yet. I thought I need to send a new file. I need to send you the changed code in the Resolve conversation?

**Comment by polyakovartem** [comment]
to understand, please read https://www.atlassian.com/git/tutorials/what-is-version-control
and https://www.atlassian.com/git/tutorials/making-a-pull-request

**Comment by alinadmytriv** [comment]
Okey, thanks!

**Pull request closed without merging**